### ISSUE TITLE
Figure out why do_stalling_workaround must be true

### ISSUE THREAD

**Issue by xftroxgpx** [xftroxgpx/a3#1]
https://github.com/xftroxgpx/a3/blob/5bd069829f56a44aecf659596d061df01019cfac/system/Z575/OSes/3archlinux/on_baremetal/filesystem_now/archlinux/home/xftroxgpx/build/1packages/chro/4_chromium-dev-git/fetch_gclient_base/checkout_root/args.gn#L2687

When false, some requests never complete. But network-wise they do! UI-wise they don't! somehow!
Requests that are non-gzip for example. (aka not `Content-Encoding: gzip`)
And only if `Content-Length` >= 1024 (requests will stall); if <=1023 will not-stall.

Only some c++ expert with quite some time on their hand can figure out this one. (I mean, I tried(as a non-expert), but gave up after a few days)

Basically it works in [the official chromium revision](https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F500808%2Fchrome-linux.zip?alt=media) that this one is based on, but whatever I changed in this one broke it! (and I did change many things, what like 14meg patch at the time, 16 now)

**Comment by xftroxgpx** [comment]
When `do_stalling_workaround    = false`
try starting chromium with:
- [ ] `--v8-cache-options=none`
- [ ] `--v8-cache-options=parse`
- [ ] `--v8-cache-options=code`

see if any have an effect on it

**Comment by xftroxgpx** [comment]
(when `do_stalling_workaround    = false`)
these `jpeg` files load and show, but disappear after pressing Stop button:
![screenshot_2017-12-30_22-05-51](https://user-images.githubusercontent.com/26243389/34457438-7ed841a8-edb0-11e7-863c-49d7e500ab71.png)
![screenshot_2017-12-30_22-06-01](https://user-images.githubusercontent.com/26243389/34457439-831b04bc-edb0-11e7-81a0-4d7efafcab7c.png)

**Comment by xftroxgpx** [comment]
Try restoring this:
https://cs.chromium.org/chromium/src/chrome/browser/chrome_content_browser_client.cc?l=3368&rcl=41afe1becca0ec4346e2f0bd0555be5351a99323
and more importantly this(which requires that^):
https://cs.chromium.org/chromium/src/chrome/browser/loader/chrome_resource_dispatcher_host_delegate.cc?l=397-413&rcl=41afe1becca0ec4346e2f0bd0555be5351a99323
